assets/sass/globals/layout.scss

```diff
@@ -2,38 +2,42 @@
    Layout Helpers
    ========================================================================= */
 
-.u-constrained {
-    max-width: $constrained;
-}
-
-.u-constrained-wide {
-    max-width: $constrainedWide;
+/**
+ * Inner/constrained layout classes
+ * .u-inner defines a constrained layout at all breakpoints
+ * .u-inner-wide-only defines a constrained layout at wider breakpoints only
+ * @TODO: Rename all uses of .inner* to u-inner*
```

Comment:
Likewise above, might as well do this while we're fixing this here? Happy to help out!